tests/config/drone/check_web_cache.sh

```diff
@@ -16,8 +16,10 @@ echo "Checking for the web cache at '$URL'."
 
 if curl --output /dev/null --silent --head --fail "$URL"
 then
-	echo "Web cache found."
+	echo "$1 cache with commit id $WEB_COMMITID already available."
 	# https://discourse.drone.io/t/how-to-exit-a-pipeline-early-without-failing/3951
 	# exit a Pipeline early without failing
 	exit 78
+else
+	echo "No $1 cache with commit id $WEB_COMMITID was available."
```

Comment:
```suggestion
	echo "$1 cache with commit id $WEB_COMMITID was not available."
```
I think this will be more clear to read. @kiranparajuli589 What do you say?